services/core-api/app/api/now_applications/response_models.py

```diff
@@ -92,7 +92,9 @@ def format(self, value):
     'NOWApplicationExplorationAccess',
     NOW_APPLICATION_ACTIVITY_SUMMARY_BASE,
     {
-       'details': fields.List(fields.Nested(NOW_APPLICATION_ACTIVITY_DETAIL_BASE,skip_none=True))
+        'has_proposed_bridges_or_culverts': fields.Boolean,
+        'bridge_culvert_crossing_description': fields.String,
+        'details': fields.List(fields.Nested(NOW_APPLICATION_ACTIVITY_DETAIL_BASE,skip_none=True))
     })
```

Comment:
`has_proposed_bridges_or_culverts` and `bridge_culvert_crossing_description` are good to be outside of `details`?